### PR TITLE
CVS-43818 Remove and resolve TODOs, adjust logging levels for pipeline runtime modifications

### PR DESCRIPTION
**Pull request by dkalinowski**

### CODE REVIEW

**Review by intel-rrozestw — APPROVED** [review]
LGTM, nice MM logger!

File: src/pipelinedefinitionstatus.hpp [hunk 1]
```diff
@@ -61,15 +61,15 @@ class MachineState {
         name(name) {}
     template <typename Event>
     void handle(const Event& event) {
-        SPDLOG_INFO("Pipeline: {} state: {} handling: {}:{}",
+        SPDLOG_DEBUG("Pipeline: {} state: {} handling: {}: {}",
```
Comment (atobiszei) [inline src/pipelinedefinitionstatus.hpp:64]: Shouldn't it be model_manager_logger?

File: src/pipelinedefinitionstatus.hpp [hunk 2]
```diff
@@ -61,15 +61,15 @@ class MachineState {
...
             name, pipelineDefinitionStateCodeToString(getStateCode()), event.name, event.getDetails());
         try {
             std::visit([this, &event](auto state) { state->handle(event).execute(*this); }, currentState);
         } catch (std::logic_error& le) {
             SPDLOG_ERROR("Pipeline: {} state: {} handling: {} error: {}", name, pipelineDefinitionStateCodeToString(getStateCode()), event.name, le.what());
             throw;
         }
-        SPDLOG_INFO("Pipeline: {} state changed to: {} after handling: {}:{}",
+        SPDLOG_INFO("Pipeline: {} state changed to: {} after handling: {}: {}",
```
Comment (atobiszei) [inline src/pipelinedefinitionstatus.hpp:72]: Same here

**Review by rasapala — APPROVED** [review]
LGTM

File: src/pipelinedefinitionstatus.hpp (hunk 1 above)
Comment (dkalinowski) [reply src/pipelinedefinitionstatus.hpp]: Done

File: src/pipelinedefinitionstatus.hpp (hunk 2 above)
Comment (dkalinowski) [reply src/pipelinedefinitionstatus.hpp]: Done

File: src/pipeline_factory.hpp [hunk 3]
```diff
@@ -70,14 +70,8 @@ class PipelineFactory {
...
         const std::vector<NodeInfo>&& nodeInfos,
         const pipeline_connections_t&& connections,
-        ModelManager& manager) {
-        auto pd = findDefinitionByName(pipelineName);
-        if (pd == nullptr) {
-            SPDLOG_ERROR("Requested to reload pipeline definition but it does not exist:{}", pipelineName);
-            return StatusCode::UNKNOWN_ERROR;
-        }
-        return pd->reload(manager, std::move(nodeInfos), std::move(connections));
-    }
```
Comment (atobiszei) [inline src/pipeline_factory.hpp:80]: Why removed?

File: src/server.cpp [hunk 4]
```diff
@@ -222,8 +222,6 @@ std::unique_ptr<ovms::http_server> startRESTServer() {
 
     auto& config = ovms::Config::instance();
     if (config.restPort() != 0) {
-        // if (config.restAddress() !=  TODO FIXME
```
Comment (atobiszei) [inline src/server.cpp:225]: Is there a follow-up ticket? I think it was @rasapala who put it here - correct?

**Review by atobiszei — APPROVED** [review]

File: src/server.cpp (hunk 4 above)
Comment (dkalinowski) [reply src/server.cpp:225]: I have contacted @intel-rrozestw who left it here and it is already addressed. We could just remove it from source.

**Pull request merged**